js/exporter/jspdf/v3/normalizeOptions.js

```diff
@@ -17,15 +27,37 @@ function normalizeBoundaryValue(value) {
     };
 }
 
-function normalizeOptions(rows) {
-    rows.forEach(row => {
+function normalizeOptions(options) {
+    // TODO: margins
+    // TODO: tableWidth
+    // TODO: tableLineWidth
+    // TODO: tableLineColor
+    // ?
+
+    if(isDefined(options.rowOptions)) {
+        const headerStyles = options.rowOptions?.headerStyles;
+        if(isDefined(headerStyles?.backgroundColor)) {
+            headerStyles.backgroundColor = normalizeColor(headerStyles.backgroundColor);
+        }
+        // TODO: groupStyles, summaryStyles, ... ???
+    }
+}
+
+function normalizeRowsInfo(rowsInfo) {
+    rowsInfo.forEach(row => {
         row.cells.forEach(({ pdfCell }) => {
             pdfCell.padding = normalizeBoundaryValue(pdfCell.padding);
-            // TODO: normalizeTextColor()
-            // TODO: normalizeBackgroundColor()
-            // TODO: ...
+            if(isDefined(pdfCell.textColor)) {
+                pdfCell.textColor = normalizeColor(pdfCell.textColor);
```

Comment:
It's not clear from tests which values can be assigned to pdfCell.textColor.
'rgb(255,255,0)' is used in some tests but it is not allowed by jsPDF docs.

http://raw.githack.com/MrRio/jsPDF/master/docs/jsPDF.html#setFillColor declares:
-  "ch1	- Number | String"
- "When only ch1 is given"
- "When only ch1,ch2,ch3 are given, "RGB" color space is implied"
- "When ch1,ch2,ch3,ch4 are given, "CMYK" color space is implied"

Imho we need tests which describes how to call each argument syntax from our customizeCell callback.
We will use these examples in our help topic for pdfCell.textColor/backgroundColor/borderColor

Reply:
> ch1,ch2,ch3

do you mean such records?:

pdfCell.textColor = '#808080'
pdfCell.textColor = 128
pdfCell.textColor = { ch1: 128, ch2: 128, ch3: 128 }
                            = { r: 128, g: 128, b: 128 }
                            = { ch1: 128, ch2: 128, ch3: 128, ch4: 1 }
                            = { r: 128, g: 128, b: 128, a: 1 }